src/components/CustomSelect/CustomSelect.scss

```diff
@@ -0,0 +1,100 @@
+@import "../../styles/variables.scss";
+
+.CustomSelect {
+  position: relative;
+
+  &-SelectedItems {
+    margin-bottom: 10px;
+    display: flex;
+    gap: 25px;
+    height: 40px;
+    &--Item {
+      padding: 8px;
+      display: flex;
+      align-items: center;
+      gap: 10px;
+      border-radius: 6px;
+      background-color: $secondary-dark-title;
+      cursor: pointer;
+      &::after {
+        content: '';
+        display: block;
+        width: 12px;
+        height: 12px;
+        background-image: url("../../assets/icons/cross.svg");
+        background-size: cover;
+        background-position: center;
+      }
+      &:hover {
+        text-decoration: line-through;
+      }
+    }
+  }
+  &-Header {
+    padding: 8px 10px;
+    display: flex;
+    justify-content: space-between;
+    border: 1px solid #A9A9A9;
+    border-radius: 6px;
+    background-color: $primary-light-bg;
+    cursor: pointer;
+    &--Placeholder {
+      // position: absolute;
+    }
+    &--Arrow  {
+      transition: 200ms ease-in-out;
+      transform: rotate(90deg);
+      svg {
+        height: 14px;
+        path {
+          fill: $primary-dark-text
+        }
+       }
+    }
+  }
+  &-Body {
+    position: absolute;
+    width: 100%;
+    left: 0;
+    top: 100%;
+    border: 1px solid #A9A9A9;
```

Comment:
border colors or all styles should be used from constants

Reply:
I agree. But variables.scss don't have needed constants. I want rework all project structure and create good scss constants